### PR TITLE
[Flutter] Fix flutter view documentation

### PR DESCRIPTION
**Pull request by peterBrxwn**

### CODE REVIEW

**Comment by peterBrxwn** [comment]
[[Flutter docs] View documentation](https://trello.com/c/rUBCKgCC/1925-flutter-docs-view-documentation)

File: flutter/current.md [hunk 1]
```diff
@@ -494,6 +494,16 @@ Timer timer = Timer(new Duration(seconds: 5), () {
 Countly.instance.sessions.updateSession();
 Countly.instance.sessions.endSession();</pre>
 <h1 id="h_01H930GAQ6R8N0G7CAPDJ60AN0">View Tracking</h1>
+<p>
+  The SDK provides access to all view-related functionality through the interface returned by:
+</p>
+<pre><code class="dart">Countly.instance.views</code></pre>
+
+<h2 id="h_01HAVQDM5TD5ZGVAC5GCABTXTF">Automatic Views</h2>
```
Comment (arifBurakDemiray) [inline flutter/current.md:502]: so for automatic views, flutter provdes those?
what about enabling it or disablng it

**Review by arifBurakDemiray — CHANGES_REQUESTED** [review]
cuple de f

File: flutter/current.md (hunk 1 above)
Comment (AliRKat) [inline flutter/current.md:502]: Agreed with Arif on this one, it should be mentioning how to enable automatic views etc. Android doc has a nice refer to this

**Review by AliRKat — CHANGES_REQUESTED** [review]

**Review by turtledreams — CHANGES_REQUESTED** [review]
we dont have autoview tracking so you can remove that. instead the intro is fine, but you can add a couple more sentence there.

File: flutter/current.md (hunk 1 above)
Comment (peterBrxwn) [reply flutter/current.md]: No option to enable or disable on flutter.

File: flutter/current.md [hunk 2]
```diff
@@ -498,8 +498,6 @@ Countly.instance.sessions.endSession();</pre>
   The SDK provides access to all view-related functionality through the interface returned by:
 </p>
 <pre><code class="dart">Countly.instance.views</code></pre>
-
-<h2 id="h_01HAVQDM5TD5ZGVAC5GCABTXTF">Automatic Views</h2>
 <p>
```
Comment (arifBurakDemiray) [inline flutter/current.md:501]: what about the body, you should also remove the body

File: flutter/next.md [hunk 3]
```diff
@@ -498,8 +498,6 @@ Countly.instance.sessions.endSession();</pre>
   The SDK provides access to all view-related functionality through the interface returned by:
 </p>
 <pre><code class="dart">Countly.instance.views</code></pre>
-
-<h2 id="h_01HAVQDM5TD5ZGVAC5GCABTXTF">Automatic Views</h2>
```
Comment (arifBurakDemiray) [inline flutter/next.md]: what about the body, you should also remove the body

Comment (peterBrxwn) [reply flutter/next.md]: alright

File: flutter/current.md [hunk 4]
```diff
@@ -494,6 +494,14 @@ Timer timer = Timer(new Duration(seconds: 5), () {
 Countly.instance.sessions.updateSession();
 Countly.instance.sessions.endSession();</pre>
 <h1 id="h_01H930GAQ6R8N0G7CAPDJ60AN0">View Tracking</h1>
+<p>
+  The SDK provides access to all view-related functionality through the interface returned by:
+</p>
+<pre><code class="dart">Countly.instance.views</code></pre>
+<p>
+  For adding segmentation to all automatic views, please refer to the
```
Comment (arifBurakDemiray) [inline flutter/current.md]: If we dont have automatic views here, we can rename the "automatic views" here to something else

File: flutter/current.md (hunk 2 above)
Comment (peterBrxwn) [reply flutter/current.md]: done

File: flutter/current.md (hunk 4 above)
Comment (peterBrxwn) [reply flutter/current.md]: Isn't this the body you suggested I remove? because I have removed it.

Comment (arifBurakDemiray) [reply flutter/current.md]: Yeap, What was that about

**Review by turtledreams — APPROVED** [review]

**Review by AliRKat — APPROVED** [review]

**Review by arifBurakDemiray — APPROVED** [review]

**Review by turtledreams — APPROVED** [review]

**Pull request merged**